.travis.yml

```diff
@@ -12,6 +12,3 @@ addons:
      - libstdc++6 # upgrade libstdc++ on linux to support C++11
 
 sudo: false
-
-after_success:
- - if [[ ${TRAVIS_NODE_VERSION} == "6" ]]; then npm run coverage; fi;
```

Comment:
what does `npm run coverage` do? (I assume it takes some time...)

Reply:
Mmm I see below. IIRC I tried to fix it and failed miserably at the time. Nobody would miss it I guess

Reply:
It used to paste an ugly message like [this one](https://github.com/CartoDB/tilelive-bridge/pull/17#issuecomment-387318060)